pennylane/ops/channel.py

```diff
@@ -0,0 +1,230 @@
+
+class PhaseDamping(Channel):
+    r"""PhaseDamping(gamma, wires)
+    Phase damping channel in the Kraus representation.
+
+    Interaction with the environment can lead to loss of quantum information changes without any
+    loss in energy. This can be modelled by the phase damping channel, with
+    the following Kraus matrices:
+
+    .. math::
+        K_1 = \begin{bmatrix}
+                0 & 0  \\
+                0 & \sqrt{\gamma}
+                \end{bmatrix}
+
+    .. math::
+        K_2 = \begin{bmatrix}
+                1 & 0 \\
+                0 & \sqrt{1-\gamma}
+                \end{bmatrix}
+
+    where :math:`\gamma` is the phase damping probability.
+
+    **Details:**
+
+    * Number of wires: 1
+    * Number of parameters: 1
+
+    Args:
+        gamma (float): phase damping probability :math:`\gamma`
+        wires (Sequence[int] or int): the wire the channel acts on
+    """
+    num_params = 1
+    num_wires = 1
+    par_domain = "R"
+    grad_method = "F"
+
+    def _kraus_matrices(cls, *params):
+        gamma = params[0]
+        if gamma > 1:
+            raise ValueError("Probabilities of noisy processes should not exceed 1.")
+        K1 = np.diag([0, np.sqrt(gamma)])
+        K2 = np.diag([1, np.sqrt(1-gamma)])
+        return [K1, K2]
+
+class DepolarizingChannel(Channel):
+    r"""DepolarizingChannel(p, wires)
+    Symmetrically depolarizing channel in the Kraus representation.
+
+    With probability :math:`p`, qubit is equally depolarized in all Pauli directions. This can
+    be modelled by the following Kraus matrices:
+
+    .. math::
+        K_1 = \sqrt{1-p} \begin{bmatrix}
+                1 & 0 \\
+                0 & 1
+                \end{bmatrix}
+
+    .. math::
+        K_2 = \sqrt{p/3}\begin{bmatrix}
+                0 & 1  \\
+                1 & 0
+                \end{bmatrix}
+
+    .. math::
+        K_3 = \sqrt{p/3}\begin{bmatrix}
+                0 & -i \\
+                i & 0
+                \end{bmatrix}
+
+    .. math::
+        K_4 = \sqrt{p/3}\begin{bmatrix}
+                1 & 0 \\
+                0 & -1
+                \end{bmatrix}
+
+
+    **Details:**
+
+    * Number of wires: 1
+    * Number of parameters: 1
+
+    Args:
+        p (float): Each Pauli gate is applied with probability :math:`\frac{p}{3}`
+        wires (Sequence[int] or int): the wire the channel acts on
+    """
+    num_params = 1
+    num_wires = 1
+    par_domain = "R"
+    grad_method = "F"
+
+    def _kraus_matrices(cls, *params):
+        p = params[0]
+        if p > 1:
+            raise ValueError("Probabilities of noisy processes should not exceed 1.")
+        K1 = np.sqrt(1-p) * np.eye(2)
+        K2 = np.sqrt(p/3) * np.array([[0, 1], [1, 0]])
+        K3 = np.sqrt(p/3) * np.array([[0, -1j], [1j, 0]])
+        K4 = np.sqrt(p/3) * np.array([[1, 0], [0, -1]])
+        return [K1, K2, K3, K4]
+
+
+ops = {"AmplitudeDamping", "GeneralizedAmplitudeDamping", "PhaseDamping", "DepolarizingChannel"}
```

Comment:
probably better to name this `__qubit_channels__` (and later we can add `__cv_channels__`)